kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -140,7 +140,7 @@ private String getRowId(SinkRecord record) {
   private RowToInsert getRecordRow(SinkRecord record) {
     return RowToInsert.of(
       getRowId(record),
-      recordConverter.convertRecord(record)
+      recordConverter.convertRecord(record, config.getBoolean(BigQuerySinkTaskConfig.CONVERT_DOUBLE_SPECIAL_VALUES_CONFIG))
```

Comment:
Can I suggest moving the shouldConvert from the method param to a constructor param for record converter? That way you can just reference it in the record converter using this.shouldConvert. Should make for cleaner code where the params of the methods aren't all getting modified.